### ISSUE TITLE
Contrast for analytics time windows is very low with the "night" theme

### ISSUE THREAD

**Issue by jgaskins** [forem/forem#13214]
The tabs to select the time frame on the analytics page look like with the dark theme on DEV has very low contrast (black on very dark blue):

**To Reproduce**

1. Set the "Night" theme
   <img width="650" alt="image" src="https://user-images.githubusercontent.com/108205/113209138-a9d80180-9240-11eb-8974-6cd0b4ab497f.png">
2. Visit [the analytics page](https://dev.to/dashboard/analytics)

**Expected behavior**

I can see all the options for the time frames to select

**Screenshots**

<img width="411" alt="" src="https://user-images.githubusercontent.com/108205/113208943-6b424700-9240-11eb-8c25-0037cefdcb7f.png">

**Desktop (please complete the following information):**

- OS, version: macOS 11.2.3
- Browser, version: Safari 14.0.3 & Firefox 86.0.1

**Comment by Link2Twenty** [comment]
Looks like it's all hard coded for these buttons. I think it just needs the CSS variables putting in (or rather `@include themeable`).

https://github.com/forem/forem/blob/95055b2a89765fde3f5a84cd33c9ed515499c2ab/app/assets/stylesheets/dashboard.scss#L282-L309

**Comment by Link2Twenty** [comment]
A question for @ludwiczakpawel , I think, should they not just be tabs? It looks like a simple class change is all that is needed to make that work. Also I added `justify-content: flex-end;` in my example. 

![asset swap](https://user-images.githubusercontent.com/3534427/113424741-9061c180-93c8-11eb-8b5b-6e0070928fdf.png)

If that is the route we take we'd need to modify a couple of things.

These classes would need updating to `crayons-tabs`, `crayons-tabs__item` and `crayons-tabs__item--current` respectively and perhaps an extra class for identification.
https://github.com/forem/forem/blob/d2d9984f28b1d0662f2a858b325a0e6b7a27a24c/app/views/shared/_stats.html.erb#L1-L5

This logic would need updating with the new classes
https://github.com/forem/forem/blob/530e9270608219e02db7774bc8e939379f22d525/app/javascript/analytics/dashboard.js#L3-L11

all of this code could be removed but we'd need to add something to shift it over to the right.
https://github.com/forem/forem/blob/95055b2a89765fde3f5a84cd33c9ed515499c2ab/app/assets/stylesheets/dashboard.scss#L282-L309

```css
.crayons-tabs.unique-class {
  justify-content: flex-end;
}
```

**Comment by ludwiczakpawel** [comment]
Analytics page is indeed outdated when it comes to UI. The ideal solution here would be to user [Crayons Tabs](https://storybook.forem.com/?path=/story/components-navigation-tabs-html--default) (even though they may not be perfectly documented, there're several places in the codebase where we use them, for example Home feed).
